stories/Components/Forms/Dropdowns/CustomSelect/CustomSelect.jsx

```diff
@@ -11,26 +11,26 @@ const CustomSelect = ({ text }) => {
       <button type="button" aria-haspopup="listbox" aria-label="Select" id="exp_button" className="selected" data-select-open="">
         {text}
       </button>
-      <ul tabIndex="-1" role="listbox" aria-labelledby="exp_button" className="options-container" data-select-options="">
+      <ul tabIndex="-1" role="listbox" aria-labelledby="exp_button" data-select-options="">
         <li id="default-language-select" className="option" role="option" data-value="default">
           <span>All</span>
         </li>
-        <li id="pasto-1" className="option" role="option" data-value="pasto">
+        <li id="pasto-1" role="option" data-value="pasto">
```

Comment:
@Ashish-srij Remove the ids from the list item here and below as well.

Reply:
done